### PR TITLE
Oel 462: facet list template overriden.

### PR DESCRIPTION
**Pull request by Maxfire**

### CODE REVIEW

File: tests/src/Kernel/SearchFacetsTest.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    'facets',
+    'facets_summary',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+    /** @var \Drupal\Core\Extension\ThemeInstallerInterface $theme_installer */
```
Comment (drishu) [inline tests/src/Kernel/SearchFacetsTest.php]: this is not needed

File: tests/src/Kernel/SearchFacetsTest.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  }
+
+  /**
+   * Tests facets summary.
+   *
+   * @covers ::setFacets
+   * @covers ::getFacets
+   * @covers ::removeFacet
+   */
+  public function testFacets() {
```
Comment (drishu) [inline tests/src/Kernel/SearchFacetsTest.php]:
```suggestion
  public function testFacetsRendering(): void {
```

File: tests/src/Kernel/SearchFacetsTest.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    $entity = new FacetsSummary(['description' => 'Owls', 'name' => 'owl'], 'facets_summary');
+
+    $this->assertEmpty($entity->getFacets());
+
+    $facets = ['foo' => 'bar'];
+    $entity->setFacets($facets);
+    $this->assertEquals($facets, $entity->getFacets());
```
Comment (drishu) [inline tests/src/Kernel/SearchFacetsTest.php]: we need to render the build of the facets summary block, and assert the markup with Crawler

**Review by drishu — CHANGES_REQUESTED** [review]
see the Sitebranding as an example or any of the other

File: templates/overrides/search/facets-summary-item-list.html.twig [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{#
+/**
+ * @file
+ * Template for a facet summary item list.
+ */
+#}
+{%- if items -%}
+  {%- for item in items -%}
+    {% if item.value['#type'] == 'link' %}
+      {% include '@oe-bcl/bcl-badge/badge.html.twig' with {
```
Comment (drishu) [inline templates/overrides/search/facets-summary-item-list.html.twig]:
```
{{ pattern('badge', {
      'label': 'label',
      'assistive_text': 'assistive_text',
      'url': '#',
      'icons_path': bcl_icon_path,
      'dismissible': true
    }) }}
```

**Review by drishu — CHANGES_REQUESTED** [review]

File: tests/src/Kernel/SearchFacetsTest.php (hunk 3 above)
Comment (Maxfire) [reply tests/src/Kernel/SearchFacetsTest.php]: Tests were excluded from this task as agreed.

File: tests/src/Kernel/SearchFacetsTest.php (hunk 2 above)
Comment (Maxfire) [reply tests/src/Kernel/SearchFacetsTest.php]: Tests were excluded from this task as agreed.

File: tests/src/Kernel/SearchFacetsTest.php (hunk 1 above)
Comment (Maxfire) [reply tests/src/Kernel/SearchFacetsTest.php]: Tests were excluded from this task as agreed.

File: templates/overrides/search/facets-summary-item-list.html.twig (hunk 4 above)
Comment (Maxfire) [reply templates/overrides/search/facets-summary-item-list.html.twig]: :+1:  done.

**Review by GilNovacomm — APPROVED** [review]
the summary title is not in the scope of this ticket, the current implementation is correct.

**Review by drishu — APPROVED** [review]

**Pull request merged**